tests/Authorization.AspNetCore.Tests/AuthorizationValidationRuleTests.cs

```diff
@@ -311,6 +311,7 @@ private static ISchema BasicSchema<T>()
     public class BasicQueryWithAttributesAndClassPolicy
     {
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0060:Remove unused parameter", Justification = "for tests")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1822:Mark members as static", Justification = "for tests")]
```

Comment:
The mark as static warning should be globally ignored.

Reply:
In general, this rule works with the exceptions of special cases.

Reply:
No. For production code it typically, but not always, makes sense. Usually it indicates where the user forgot to mark a method as virtual more than it marks where a method can be made static. This is a dumb analyzer that is not taking the code into context when considering the recommendation. Static methods should be helper functions and similar, not methods that provide class specific logic flow. (Perhaps that’s my opinion however.)

When it comes to tests there is often no reason to mark methods as static or virtual. Marking these methods either way is entirely non beneficial.

Reply:
ignored